frame/fast-unstake/src/lib.rs

```diff
@@ -192,6 +201,29 @@ pub mod pallet {
 				return Weight::from_ref_time(0)
 			}
 
+			// Temporary for migration.
+			if Head::<T>::exists() && Head::<T>::get().is_none() {
+				// Migrate head.
+				let key = Head::<T>::hashed_key();
+				let (stash, deposit) = match frame_support::storage::unhashed::get::<(
+					T::AccountId,
+					Vec<EraIndex>,
```

Comment:
This had type:  
```suggestion
					BoundedVec<EraIndex, MaxChecked>,
```

Reply:
Bounds don't have any impact on the decoding of a type, but yeah I can add it back.